linera-core/src/client.rs

```diff
@@ -203,14 +209,33 @@ pub enum MessagePolicy {
 }
 
 impl MessagePolicy {
+    pub fn new(
+        blanket: BlanketMessagePolicy,
+        restrict_chain_ids_to: Option<HashSet<ChainId>>,
+    ) -> Self {
+        Self {
+            blanket,
+            restrict_chain_ids_to,
+        }
+    }
+
+    #[tracing::instrument(level = "trace", skip(self))]
+    fn accepts(&self, bundle: &IncomingBundle) -> bool {
```

Comment:
This should probably implement the whole policy (including "blanket") then `is_ignore` and `is_reject` might be removed.

Reply:
I think this is a bit more involved so I will leave it as a separate PR.